nni/algorithms/hpo/random_tuner.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import numpy as np
+import schema
+
+from nni import ClassArgsValidator
+from nni.common.hpo_utils import format_search_space, deformat_parameters
+from nni.tuner import Tuner
+
+class RandomTuner(Tuner):
```

Comment:
does this tuner still support nested search space?

Reply:
I believe so.